libs/transit/transit_serdes.hpp

```diff
@@ -35,38 +33,48 @@ double constexpr kMinDoubleAtTransitSection = kInvalidWeight;
 double constexpr kMaxDoubleAtTransitSection = 10000000.0;
 uint8_t constexpr kDoubleBits = 32;
 
-template <typename Sink>
+namespace serdes
+{
+template <typename T>
+concept DefaultIntegral =
+    std::is_same<T, bool>::value || std::is_same<T, uint8_t>::value || std::is_same<T, uint16_t>::value;
```

Comment:
I don't see the point in writing more text. We have an alias
(yes, std::size_t is different)
```suggestion
    std::is_same_v<T, bool> || std::is_same_v<T, uint8_t> || std::is_same_v<T, uint16_t>;
```